frontend/public/components/dashboard/events-card/events-card.scss

```diff
@@ -40,12 +40,6 @@
   white-space: nowrap;
 }
 
-.co-events-card__item-source {
```

Comment:
since you're completely removing this rule, you should also remove it from https://github.com/openshift/console/blob/master/frontend/public/components/dashboard/events-card/event-item.tsx#L33

Reply:
Not familiar with TypeScript at all, but I removed `className` entirely, hopefully that's right. 🙂

Reply:
you did well :)